variables.tf

```diff
@@ -319,3 +319,33 @@ variable "volume_tags_enabled" {
   default     = true
   description = "Whether or not to copy instance tags to root and EBS volumes"
 }
+
+variable "ssm_patch_manager_enabled" {
+  type        = bool
+  default     = false
+  description = "Weather to enable SSM Patch manager"
+}
+
+variable "ssm_patch_manager_patch_tag" {
```

Comment:
are these tag vars in use?

Reply:
this are additional tags that the instance needs to have so I could merge them to the tag var.